app/_locales/en/messages.json

```diff
@@ -1910,10 +1910,10 @@
     "message": "Show"
   },
   "showAdvancedGasInline": {
-    "message": "Advanced gas controls"
+    "message": "Advanced Fee Controls"
```

Comment:
I personally like this change. What do you think @danjm @darkwing @ryanml @NiranjanaBinoy